client/electron/src/setup.ts

```diff
@@ -104,46 +104,51 @@ export class ElectronCapacitorApp {
     this.config = newConfig;
 
     this.TrayMenuTemplate = [
+      new MenuItem({
+        label: this.config.hasOwnProperty('locale') && (this.config as any).locale === 'fr-FR' ? 'Afficher Parsec' : 'Show Parsec',
+        click: () => {
+          this.showMainWindow();
+        },
+      }),
       new MenuItem({
         label: this.config.hasOwnProperty('locale') && (this.config as any).locale === 'fr-FR' ? 'Quitter' : 'Quit',
         click: () => {
-          this.forceClose = true;
-          app.quit();
+          this.showMainWindow();
+          this.MainWindow.webContents.send('close-request');
         },
       }),
     ];
     this.TrayIcon.setContextMenu(Menu.buildFromTemplate(this.TrayMenuTemplate));
   }
 
-  getTray(): boolean {
+  isTrayEnabled(): boolean {
     if ('minimizeToTray' in this.config) {
       return (this.config as any).minimizeToTray;
     }
     return false;
   }
 
   hideMainWindow(): void {
-    if (process.platform === 'win32') {
-      this.MainWindow.hide();
-    } else {
+    if (process.platform === 'darwin') {
       this.MainWindow.minimize();
+    } else {
+      this.MainWindow.hide();
     }
   }
 
   showMainWindow(): void {
-    if (process.platform === 'win32') {
-      this.MainWindow.show();
-    } else {
+    if (process.platform === 'darwin') {
       this.MainWindow.restore();
+    } else {
+      this.MainWindow.show();
     }
-    this.MainWindow.focus();
   }
 
   isMainWindowVisible(): boolean {
-    if (process.platform === 'win32') {
-      return this.MainWindow.isVisible();
+    if (process.platform === 'darwin') {
+      return this.MainWindow.isMinimized();
     } else {
-      return !this.MainWindow.isMinimized();
+      return !this.MainWindow.isVisible();
```

Comment:
conditions seems to be reversed, it should probably be:
```
    if (process.platform === 'darwin') {
      return !this.MainWindow.isMinimized();
    } else {
      return this.MainWindow.isVisible();
    }
```

Reply:
Nice catch !